apps/pages/models.py

```diff
@@ -28,7 +28,7 @@ class Page(models.Model):
     )
     author = models.CharField(max_length=256, blank=True)
     url = models.CharField(max_length=128, db_index=True, validators=[validate_page_url])
-    category = models.ForeignKey(Category, on_delete=models.SET_NULL, null=True)
+    category = models.ForeignKey(Category, on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
Just to note - you might need another booleanfield on the model, `show_in_navigation` - or something similar.

Otherwise this thank you page will appear in the list of visible pages!

Reply:
My idea was if the page doesn't have the category won't appear anything let's got with this for now.

Reply:
Not a bad idea! Maybe add `help_text` to explain that